samples/bluetooth/central_and_pripheral_hr/README.rst

```diff
@@ -0,0 +1,170 @@
+.. |sample path| replace:: :file:`samples/bluetooth/central_and_peripheral_hr`
+
+.. include:: /includes/build_and_run.txt
+
+
+.. _central_and_peripheral_hrs_testing:
+
+Testing
+=======
+
+After programming the sample to your development kit, you can test it either by connecting to another development kits that are running the :ref:`peripheral_hids_keyboard` sample, or by using `nRF Connect for Desktop`_ that emulates a HRS server.
+
+Testing with another development kits
+-------------------------------------
+
+1. |connect_terminal|
+#. Program the other kit with :ref:`zephyr:peripheral_hr` samples and reset it.
+#. Wait until the HRS is detected by the central.
+   Observe that **LED 2** is on.
+#. In the terminal window, check for information similar to the following:
+        Heart Rate Sensor body location: Chest
+        Heart Rate Measurement notification received:
+
+            Heart Rate Measurement Value Format: 8 - bit
+            Sensor Contact detected: 1
+            Sensor Contact supported: 1
+            Energy Expended present: 0
+            RR-Intervals present: 0
+
+            Heart Rate Measurement Value: 134 bpm
+
+   Notifications will be displayed periodically with a frequency determined by the HR server.
+
+#. Program another kit with :ref:`zephyr:bluetooth_central_hr` and reset it.
+#. Wait until central is connected to your development kit.
+   Observe that **LED 3** is on.
+#. In terminal windows connected to :ref:`zephyr:bluetooth_central_hr`, check for information similar to following:
+
+        [NOTIFICATION] data 0x20006779 length 2
+
+#. Sample is working as relay now.
+
+Testing with nRF Connect for Desktop
+------------------------------------
+1. |connect terminal|
+#. Reset the kit.
+#. Start `nRF Connect for Desktop`_ and select the connected dongle that is used for communication.
+#. Go to the :guilabel:`Server setup` tab.
+   Click the dongle configuration and select :guilabel:`Load setup`.
+   Load the :file:`hr_service.ncs` file that is located under :file:`samples/bluetooth/central_and_peripheral_hr` in the |NCS| folder structure.
+#. Click :guilabel:`Apply to device`.
+#. Go to the :guilabel:`Connection Map` tab.
+   Click the dongle configuration and select :guilabel:`Advertising setup`.
+
+   The current version of nRF Connect can store the advertising setup.
+
+   #. Click :guilabel:`Load setup`.
+      Load the :file:`hrs_adv_setup.ncs` file that is located under :file:`samples/bluetooth/central_and_peripheral_hr` in the |NCS| folder structure.
+   #. Click :guilabel:`Apply` and :guilabel:`Close`.
+
+#. In the Adapter settings, choose :guilabel:`Start advertising`.
+#. Wait until the kit that runs the Central and Peripheral HRS connects.
+    Observe that **LED 2** is on.
+#. Explore the :guilabel:`Heart Rate Measurement` characteristic.
+   Write value `06 80` and click :guilabel:`Play` button to send notification.
+   In the terminal window, check for information similar to the following:
```

Comment:
```suggestion
   Write value ``06 80`` and click the :guilabel:`Play` button to send a notification.
   In the terminal window, check for information similar to the following:
```
Would this be a step as well, or is it explaining how to explore the characteristic?